distil/primitives/satellite_image_loader.py

```diff
@@ -150,21 +150,6 @@ class DataFrameSatelliteImageLoaderPrimitive(
                         git_commit=utils.current_git_commit(os.path.dirname(__file__)),
                     ),
                 },
-                {
```

Comment:
Does pip installing the python lz4 library ensure that any accompanying C code is built and installed as well?  If not, then you need to ensure that the version of Ubuntu that D3M uses (18.04 I think) has the necessary system libraries in it already.  For the original lzo library, the section below was responsible for ensuring that the d3m build process would install the necessary libraries as they weren't part of the Ubuntu image.  The pip installation was required in the same section as it respect order, and ensures the necessary system libraries are installed before the pip install of the python library.  If it is left to `setup.py`, you will end up installing the python library before the dependencies, which may fail.